grobid-core/src/main/java/org/grobid/core/main/LibraryLoader.java

```diff
@@ -158,7 +158,9 @@ public boolean accept(File dir, String name) {
                     if (StringUtils.isNotEmpty(GrobidProperties.getPythonVirtualEnv())) {
                         String virtualEnv = GrobidProperties.getPythonVirtualEnv() + File.separator + "lib";
 
-                        List<Path> pythons = Files.find(Paths.get(virtualEnv), 1, (path, attr) -> path.getFileName().toString().startsWith("libpython3.6m")).collect(Collectors.toList());
+                        List<Path> pythons = Files.find(Paths.get(virtualEnv), 1, (path, attr) -> path.getFileName().toString().startsWith(
+                            "python3.6"
```

Comment:
this needs to be more flexible, maybe remove filtering?

Reply:
DeLFT is tested with python 3.5, I have not considered 3.6 when integrating with GROBID. To support different version of python, we would need to ship different version of JEP binaries and adapt the library path to be added based on the python version used to build DeLFT.  

This filtering part has been written by Luca, I've not looked at it yet!

Reply:
I have changed that to _contains_ `python3` for now.